sparkmagic/sparkmagic/tests/test_livyreliablehttpclient.py

```diff
@@ -60,3 +63,11 @@ def test_get_all_session_logs():
     assert_equals(out, http_client.get.return_value.json.return_value)
     http_client.get.assert_called_once_with("/sessions/42/log?from=0", [200])
 
+def test_custom_headers():
+    custom_headers = {"header1": "value1"}
+    overrides = { conf.custom_headers.__name__: custom_headers }
+    conf.override_all(overrides)
+    endpoint = Endpoint("http://url.com", constants.NO_AUTH)
+    client = LivyReliableHttpClient.from_endpoint(endpoint)
+    headers = client.get_headers()
+    assert_equals(custom_headers.items() <= headers.items(), True)
```

Comment:
Please check that the content type header is present explicitly and that the header1 header is also there. And that there is only two headers. You can remove the assert you have now. I think it's too weak of an assertion.